sigma/sigma_mt.c

```diff
@@ -25,11 +25,18 @@
 
    mt           7320 and 7322/7323 magnetic tape
 
+   11-Feb-24    RMS     Report non-operational if not attached (Ken Rector)
+   01-Feb-24    RMS     Fixed nx unit test (Ken Rector)
+   01-Nov-23    RMS     Fixed reset not to clear BOT
+   31-Mar-23    RMS     Mask unit flag before calling status in AIO (Ken Rector)
+   07-Feb-23    RMS     Silenced Mac compiler warnings (Ken Rector)
+   15-Dec-22    RMS     Moved SIO interrupt test to devices
    20-Jul-22    RMS     Space record must set EOF flag on tape mark (Ken Rector)
    03-Jul-22    RMS     Fixed error in handling of channel errors (Ken Rector)
    02-Jul-22    RMS     Fixed bugs in multi-unit operation
    07-Jun-22    RMS     Removed unused variables (V4)
    26-Mar-22    RMS     Added extra case points for new MTSE definitions
+   23-Mar-20    RMS     Unload should call sim_tape_detach (Mark Pizzolato)

```

Comment:
Is this only a comment change?  Please confirm that the actual code change was in fact made before March 2022.
By the way, it would be more logical for changes to be marked with the initials of the person making the change, rather than by RMS, if Bob was not that person.

Reply:
What simulator is this for? This PR does not follow conventions.

Reply:
@pkoning2 - I looked back in the sigma_mt.c history and found that the unload/detach code change was made in a commit of files for several simulators by Mark P on April 4, 2020.  Mark did not add a comment like this at that time.  

I think Mark did not include these kinds of commemts, but Bob did in his classic simh archive.  So, in my adding new code from Bob this comment slipped in.  Mark did include comments of this type in later commits.

I could delete this comment or change its date to be more accurate or do nothing.
Your choice.

Bob did make these changes.  

Usually I find a problem, do some research and give him a description of the problem and any ideas I have about how to fix it. Usually my fix works for me and Bob will consider that and come up with suitable code.  I think he puts my name on it out of courtesy.  If you don't like that, its not necessary by me, I have enough noteriety.

@rcornwell - Where could I find the PR conventions?  


I'm eager to fix this.  Can you get me started with how to do that.

Why did you both reference the sigma_mt.c changes?

Why did this fail two out of six build tests and what could I do to fix that?

Reply:
Ken,
Thanks for the confirmation.  With that, I have no concern about the change.  I mentioned sigma_mt.c specifically because that's where I saw the added comment referring to Mark, and that made me want to verify we didn't have a  problem with the policy restricting changes imported from Mark's "simh" repository.

On the test failure, that's a machinery issue not related to your work.  I don't know why this happens; it seems to be intermittent.  You may ignore it.